src/nimblepkg/version.nim

```diff
@@ -17,32 +17,32 @@ type
     verAny, # *
     verSpecial # #head
 
-  PVersionRange* = ref TVersionRange
-  TVersionRange* = object
-    case kind*: TVersionRangeEnum
+  VersionRangeRef* = ref VersionRange
+  VersionRange* = object
```

Comment:
You shouldn't make the non-ref type accessible as it is not used anywhere from what I can see and remember. So we should just have a ``VersionRange``.